docs/boilerplate/app/theme/Theming.md

```diff
@@ -31,3 +31,151 @@ Timings are defined in `app/theme/timing.ts`. They can be used for consistent an
 Spacing is a first class citizen in Ignite. We use a spacing scale to define the spacing between elements in the app. This allows us to have a consistent spacing scale across the app, and also allows us to change the spacing easily. It is recommended to use the spacing scale for all spacing in the app if possible.
 
 [Spacing](./spacing.ts.md)
+
+# Multiple Themes (a.k.a "Dark Mode")
+
+The Ignite boilerplate ships with color palette definitions and support for multiple themes! By default we define two themes, but you can easily add more using our generic theming system.
+
+:::tip
+
+Head on over to the [Ignite Cookbook](https://ignitecookbook.com/) to find recipes for how to integrate Ignite's theming system with other popular styling and component libraries!
+
+:::
+
+In Ignite `<10` we relied on pure style objects typed as `ViewStyle`, `TextStyle`, `ImageStyle`, etc. This didn't quite fit circumstances where you wanted to dynamically change the overall app's theme during runtime. You could always reload the javascript bundle to see changes, but that was a less than ideal solution.
+
+Ignite now has the concept of `ThemedStyle`s. You can stick with the traditional pure style objects but if you want to take advantage of themed global colors, spacing, typography, and timings, you've got to add a little more code...
+
+Here's an example of how to use the new `useAppTheme()` hook to style your components and dynamically use the correct theme:
```

Comment:
```suggestion
**New in Ignite 10**: the `useAppTheme` hook allows you to create dynamically-themed styles, right out of the box. Here's an example:
```

Rationale for suggestion: most people are probably not going to be using Ignite multiple times or paying attention to versions; I think it's sufficient to explain that there's now a new feature and how to use it.